project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.features.field_base.inc

```diff
@@ -82,6 +82,25 @@ function c4m_content_group_field_default_field_bases() {
     'type' => 'entityreference',
   );
 
+  // Exported field_base: 'c4m_group_welcome_message'.
+  $field_bases['c4m_group_welcome_message'] = array(
```

Comment:
@anvmn not if they are only used in one content type (which is the case here), so they can stay here!